packages/react/focus-scope/src/FocusScope.tsx

```diff
@@ -103,7 +110,8 @@ const FocusScope = React.forwardRef((props, forwardedRef) => {
   React.useEffect(() => {
     if (container) {
       focusScopesStack.add(focusScope);
-      const PREVIOUSLY_FOCUSED_ELEMENT = document.activeElement as HTMLElement | null;
```

Comment:
Never understood why we uppercase this, we don't do this for any of our other variables.

Reply:
This bothered me as well haha.

Reply:
haha no reason whatsoever, no idea why I did that 😀